SnoopCompileBot/src/snoop_bench.jl

```diff
@@ -4,15 +4,15 @@ function _snoopi_bench_cmd(snoop_script)
     return quote
         global SnoopCompile_ENV = true
 
-        using SnoopCompileCore
+        using SnoopCompile
```

Comment:
This reverts the package to the pre-split era. I wanted to avoid doing this in #112. Now online usage of SnoopCompile will give the wrong results.

Reply:
Why? It's just a question of whether you load SnoopCompile + bot, or just the bot.

But there really isn't a choice: if you wanted it this way we needed to call that release 2.0. We can move there, but I was trying to maintain backwards compatibility.

Reply:
Here we are loading the SnoopCompile internally, which loads all the packages. So it does not matter what the user wants to do. 

The problem of the bot breaking in version 1.6.0 and 1.6.1 is that Pkg cannot handle adding these to the global environment. That is why we need to **add them before using** like how I did in #112. There is no difference between SnoopCompile or SnoopCompileBot. That is why CI fails here.
https://github.com/timholy/SnoopCompile.jl/pull/113/checks?check_run_id=816357714#step:6:205